README.md

```diff
@@ -106,6 +76,79 @@ These steps you need to do once to setup the project.
+should be already be up. You can check with 
+
+  ```bash
+  docker ps -a
+  ```
+
+which should give you a result like this : 
+  ```
+  CONTAINER ID   IMAGE                  COMMAND                   CREATED        STATUS                      PORTS                     NAMES
+  a63582388800   minio/mc               "/bin/sh -c '\n  set …"   39 hours ago   Exited (0) 40 seconds ago                             service-stac_s3-client_1
+  33deededf690   minio/minio            "/usr/bin/docker-ent…"    39 hours ago   Up 41 seconds               0.0.0.0:9090->9000/tcp    service-stac_s3_1
+  d158be863ac1   kartoza/postgis:12.0   "/bin/sh -c /docker-…"    39 hours ago   Up 41 seconds               0.0.0.0:15432->5432/tcp   service-stac_db_1
+  ```
+
+As you can see, MinIO is using two containers, one is the local S3 server, the other is a S3 client used to initialise
+some variables to allowe MinIO to run smoothly (and exited normally afterwards). You should also have a postGIS container.
+
+
+Another way to start these containers (if, for example, they stopped) is with a simple
+
+  ```bash
+  docker-compose up
+  ```
+
+in the source root folder (this is the last command of `make setup`). Make sure to run `make setup` before to ensure the necessary folders `.volumes/*` are in place. These folders are mounted in the services and allow data persistency over restarts of the containers.
+
+If you want to use a local postgres instance instead of the dockerised one you need additionally
+
+To use a local postgres instance rather than a container, once you've ensured you've the needed dependencies, you should : 
+
+- Create a new superuser (required to create/destroy the test-databases) and a new database.
+
+*Note: the user/password and database name in the example below can be changed if required, these names reflects the one in `.env.default`.*
+
+```bash
+sudo su - postgres
+psql
+# create a new user, for simplicity make it a superuser
+# this allows the user to automatically create/destroy
+# databases (used for testing)
+psql> CREATE USER service_stac WITH PASSWORD 'service_stac';
+psql> ALTER ROLE service_stac WITH SUPERUSER;
+# We need a database with utf8 encoding (for jsonfield) and utf8 needs template0
+psql> CREATE DATABASE service_stac_local WITH OWNER service_stac ENCODING 'UTF8' TEMPLATE template0;
+```
+
+The PostGIS extension will be installed automatically by Django.
+
+**Note: this is a local development setup and not suitable for production!**
+
+You might have to change your .env.local file especially the DB_PORT, if you're using this setup.
+
+Lastly, once your databases have been set up, it is time to apply migrations (to have the latest model) and fill it with
+some default values to be able to start working with it. (From the root)
+  ```bash 
+  pipenv shell
+  ./app/manage.py migrate
+  ./app/manage.py populate_testdb
+  ```
+
+the ```pipenv shell``` command activate the virtual environment provided by pipenv.
```

Comment:
This should be moved in the ending of chapter `### Setting up the local database` before new chapter for using local postgres